src/pages/workspace/withPolicyAndFullscreenLoading.js

```diff
@@ -1,8 +1,8 @@
-import isEmpty from 'lodash/isEmpty';
 import omit from 'lodash/omit';
 import PropTypes from 'prop-types';
 import React from 'react';
 import {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
```

Comment:
I thought we were deprecating `underscore`. Can we just import it from `lodash` instead?